examples/github-with-inline-customizations/main.tf

```diff
@@ -0,0 +1,61 @@
+terraform {
+  required_version = "= 1.7.0"
+
+  required_providers {
+    flux = {
+      source  = "fluxcd/flux"
+      version = "1.2.3"
```

Comment:
Can we use semver range here instead of pinning the versions, we don't want to keep bumping these all the time.

Reply:
Changes made.